guides/hack/21-XHP/01-setup.md

```diff
@@ -1,13 +1,41 @@
-## Namespace Support
-
-Support for namespaced XHP classes (elements) is enabled by default since
+Support for XHP and namespaced XHP classes (elements like `<p>`) is enabled by default since
 HHVM 4.73. It comes with many related
 [XHP changes](https://hhvm.com/blog/2020/09/02/XHP-namespaces-and-syntax.html)
 compared to previous versions.
 
-We recommend using this version or newer, since it's more thoroughly
-tested and doesn't require any extra configuration, but XHP namespace support
-can also be enabled in older HHVM versions (since around HHVM 4.46) by adding
+## The XHP-Lib Library
+
+While the XHP syntax is part of Hack, a large part of the implementation is in a
+library called [XHP-Lib](https://github.com/hhvm/xhp-lib/) that needs to be
+installed via [Composer](https://getcomposer.org/) (add `facebook/xhp-lib` to your `composer.json` manually
+or by running `composer require facebook/xhp-lib ^4.0` or `^3.0`).
+
+This includes the base classes and interfaces, and definitions of standard HTML elements.
+
+### XHP-Lib Version
+
+There are currently two major supported versions of XHP-Lib:
```

Comment:
Facebook stopped supporting v3 a while ago. It might be a good idea to reflect this in the docs. @fredemmott knows the state of v3. My interpretation of the project status is: Facebook does not recommend you use v3 as it requires non-default hhvm ini settings. V4 is strongly recommended. If v3 falls over in newer hhvm versions, it is up to the community to open a PR and to request a new release be made.

Reply:
Added a note on the status of both libraries.